run.py

```diff
@@ -0,0 +1,123 @@
+import torch
+
+import spikeinterface as si
+import spikeinterface.sorters as ss
+
+import os
+import sys
+import zipfile
+import boto3
+
+from kilosort import io, run_kilosort
+from spikeinterface.extractors import read_nwb_recording, read_maxwell
+
+
+this_dir = os.path.dirname(__file__)
+output_dir = os.path.join(this_dir, 'output')
+
+
+def zip_files(dir: str, zip_filename: str):
+    with zipfile.ZipFile(zip_filename, 'w') as z:
+        for root, dirs, files in os.walk(dir, topdown=False):
+            for f in files:
+                f = os.path.join(root, f)
+                z.write(f, os.path.basename(f))
+
+
+def mock_run_sorter(input_filepath: str):
```

Comment:
Might add a comment explaining the purpose of this method and how it fits into the testing scheme.